template_rpm/3rd_party_software/RPM-GPG-KEY-rpmfusion-free-fedora-36

```diff
@@ -0,0 +1,29 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
Those were symlinks to the actual keys (the same for multiple releases). Any specific reason for committing separate copies?

Reply:
No, it's certainly a failure during backporting or similar.